src/middlewares/task.middleware.js

```diff
@@ -119,9 +120,7 @@ export default (store: any) => (next: any) => (action: any) => {
         ['run', name, ...additionalArgs],
         {
           cwd: projectPath,
-          env: {
-            ...window.process.env,
-          },
+          env: getBaseProjectEnvironment(projectPath),
```

Comment:
@superhawk610 I think it's OK to add the path to `./node_modules/.bin` here but I think that shouldn't be needed. Maybe we can check this later and leave this for now.

It should be enough to have `cwd` as it is like `cd` to project directory and run `yarn` inside that folder. Then yarn will look into `./node_modules/` for the file to execute and if is not finding the file to executable it will look if it's globally available.

Reply:
Without explicitly providing the `.bin` path, I get some form of this error on Windows:

```
$ react-scripts test --env=jsdom --coverage
'react-scripts' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
Task completed
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

It may be a Git Bash-specific issue.